test/libsolidity/Metadata.cpp

```diff
@@ -552,15 +530,6 @@ BOOST_AUTO_TEST_CASE(metadata_license_natspec_multiline)
 	BOOST_CHECK(compileAndCheckLicenseMetadata("C", sourceCode) == "GPL-3.0");
 }
 
-BOOST_AUTO_TEST_CASE(metadata_license_no_whitespace)
-{
-	char const* sourceCode = R"(
-		//SPDX-License-Identifier:GPL-3.0
-		contract C {}
-	)";
-	BOOST_CHECK(compileAndCheckLicenseMetadata("C", sourceCode) == "GPL-3.0");
-}
-
```

Comment:
Was this removed because we already have this test in a .sol file? Note that the .sol file has a comment:
```
// This test is actually useless, as the test suite adds the automatic preamble.
```

See https://github.com/ethereum/solidity/pull/10653#discussion_r545974629 and https://github.com/ethereum/solidity/pull/10653#discussion_r558422143.

Reply:
I am actually not sure anymore at this point. I added it back and it works. I suspect the whitespace parts were not working at some point during this PR and only successful tests are useful in the Metadata.cpp